content/working_with/console/pages/dashboard-page.md

```diff
@@ -14,7 +14,7 @@ The Dashboard page is the landing page when you log into the {{< param cfy_conso
 
 ### Statistics
 
-By default, the page displays usage statistics of the {{< param cfy_manager_name >}}, including number of services and number of Kubernetes clusters on this {{< param cfy_manager_name >}}.
+By default, the page displays usage statistics of the {{< param cfy_manager_name >}}, including number of: services, Kubernetes clusters and Terraform modules on this {{< param cfy_manager_name >}}.<br />
```

Comment:
As far as I know, it doesn't work like that 😁    

We can just <click enter> to create a new line, but the markdown won't interpret it as we'd guess 😁     
To create a new line, there are 3 known to me ways:
- Use some sort of the block element, which automatically is going into the another line (e.g. this bullet point 🚀 ),
- Write 2 (or more) whitespaces, then click enter
- Use `<br />` tag

Before commiting changes I was going with the double whitespace convention, but then I've recalled that once we had a problem with that, when my IDE changed some of the whitespaces inside a similar markdown file.
Because of that, I've went with the `<br />` approach, to implicitly show when the new line should be created

Additionally, I'm attaching a fresh example of just clicking the <enter> button, to make a new, visible endlines in the documentation:
![new line](https://user-images.githubusercontent.com/89005877/150975451-3b7da440-a5f0-4a8e-950d-b8b738768f6c.png)

Cause, it's always good to test the hypothesis 😁

Reply:
I was referring to empty line 🙂 

See e.g. here in **Blueprint details** section - first two paragraphs are separated. It comes from having empty line in the source file:
![image](https://user-images.githubusercontent.com/5202105/150978143-59e199d0-11f3-49e1-adce-6e92117150ff.png)

Reply:
Ouuu, I didn't knew that, thanks for the tip!
I've just did a quick test and it seems like that action is indeed adding a new line, but it's also breaking the bullet list styling
See the example:
![new line_2](https://user-images.githubusercontent.com/89005877/150997395-8f9f4980-8b23-4fc5-9c7d-f6b1dbec4351.png)